### ISSUE TITLE
responsive chartjs

### ISSUE THREAD

**Issue by SimonErich** [aomra015/ember-cli-chart#10]
Hi I'm trying to integrate the addon in my dashboard and can't get them to work responsively.
It should fit to their corresponding div in width and height (on init and on resizing). any ideas?

**Comment by aomra015** [comment]
@SimonErich let me know if this issue helps: https://github.com/nnnick/Chart.js/issues/56

This component is simply using ChartJS so most issues will be resolved by going to the source.

**Comment by sereysethy** [comment]
I think there was something wrong in the way you create Chartjs or the library chartjs itself. If I included the `canvas` element directly into the view/template, the chart is displayed responsively fine. But when I use your component `ember-chart`, the chart is small, it does not scale as its parent width and height. I even put two charts side by side. The one with `canvas` element directly inserted into the view is display fine (I took the example from chartjs website).

I attached the image for you.
![screen shot 2016-03-28 at 5 01 20 pm](https://cloud.githubusercontent.com/assets/12994180/14075841/dacccf8c-f506-11e5-8275-8a5d12a88e45.png)
